pkg/postage/mock/service.go

```diff
@@ -93,7 +94,20 @@ func (m *mockPostage) HandleCreate(_ *postage.Batch) error { return nil }
 
 func (m *mockPostage) HandleTopUp(_ []byte, _ *big.Int) {}
 
-func (m *mockPostage) HandleDepthIncrease(_ []byte, _ uint8, _ *big.Int) {}
```

Comment:
So this is a mock implementation. This is created mainly for using in tests. If we start adding code here for features for dev mode, we will need to start writing tests for these separately.

I believe this bug for dev mode should not be fixed at all.

Reply:
@istae @aloknerurkar how should I proceed with from here?

Reply:
So if you want to provide the functionality for dev mode, I would suggest either implementing a different package which does things similar to mock but it has unit tests for the entire code.

Then we can use this pkg instead of the mock one in dev mode init. The same goes for the other PR.